src/insights_client/__init__.py

```diff
@@ -398,13 +421,17 @@ class InsightsConstants(object):
 
         # handle client instantation here so that it isn't done multiple times in __init__
         # The config can be passed now by parameter
+        logger.debug("Initializing InsightsClient with the loaded config...")
         client = InsightsClient(config, False)  # read config, but dont setup logging
+        logger.debug("InsightsClient initialized. Egg version: %s" % (client.version(),))
 
         # we now have access to the clients logging mechanism instead of using print
+        logger.debug("Setting up regular logging.")
+        tear_down_logging()
         client.set_up_logging()
-        logging.root.debug("Loaded initial egg: %s", os.path.dirname(insights.__file__))
 
         for p in get_phases():
+            logger.debug("Running phase %s." % (p["name"],))
```

Comment:
Nitpick: Why do you keep mixing two different ways of formatting things?
```py
logger.debug("Running phase %s." % (p["name"],))
logger.debug("Running phase %s.", p["name"])

logger.debug("Running phase '{phase}'.".format(phase=p["name"]))
```

Reply:
I wanted to unify this, leveraging the native logging interpolation. For some reason, I didn’t do it consistenly. Good catch! 🙇🏻‍♂️